mod/missions/views/default/forms/missions/sort-missions-form.php

```diff
@@ -141,4 +142,42 @@
         </div>
 		
 	</div>
+    
+    <div class="col-sm-11 mrgn-tp-lg">
```

Comment:
Can this mrgn-tp-lg only be added when the filter string. Right now there is a large blank space when I am not filtering anything.